.env.example

```
@@ -16,4 +16,9 @@
 
 # ── Server ──────────────────────────────────────────────────────────────────
 # FLASK_PORT=5000
-# FLASK_DEBUG=false
\ No newline at end of file
+# FLASK_DEBUG=false
+
+# ── Docker Setup ────────────────────────────────────────────────────────────
+# If using docker-compose you can pass the image tag via TAG env var:
+# TAG=v1.2.3  docker compose up -d
+# Defaults to "latest" when not set.
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)